### PR TITLE
[RELEASE] 20170607

### PR DESCRIPTION
**Pull request by MiguelMoll**
## Pull Requests
  - closes #2217 Reliable build export [@MiguelMoll]
  - closes #2219 ami updates [@MiguelMoll]

## Milestone Release
- [x] Release branch
- [x] Pass CI
- [x] Code review
- [x] Merge into master
- [x] Close milestone
- [x] Release master
- [ ] Record release number: 
- [ ] Pass CI
- [ ] Update demo
- [ ] Deploy demo/httpd
- [ ] Deploy demo/rails
- [ ] Update staging
- [ ] Deploy staging/site-staging
- [ ] Deploy staging/console-staging
- [ ] Verbose [release notes](https://github.com/convox/rack/releases)
- [ ] Documentation review
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[20170607](https://trello.com/c/ZE8hoSXn/2541-20170607)

File: provider/aws/builds.go [hunk 1]
```diff
@@ -194,56 +194,77 @@ func (p *AWSProvider) BuildExport(app, id string, w io.Writer) error {
...
+	}
 
-		header := &tar.Header{
-			Typeflag: tar.TypeReg,
-			Name:     fmt.Sprintf("%s.%s.tar", service, build.Id),
-			Mode:     0600,
-			Size:     stat.Size(),
-		}
+	name := fmt.Sprintf("%s.%s.tar", app, build.Id)
+	file := filepath.Join(tmp, name)
```
Comment (nzoschke) [inline provider/aws/builds.go]: is there a disk usage or cleanup concern here?

Comment (nzoschke) [inline provider/aws/builds.go]: I see a remove later, but should that be a defer()?

Comment (MiguelMoll) [inline provider/aws/builds.go]: Good point. Cleanup is now a defer.

**Comment by MiguelMoll** [comment]
Not publishing as the AMIs have been reverted.

**Pull request merged**